btf/traversal.go

```diff
@@ -101,12 +101,24 @@ func walkType(typ Type, fn func(*Type)) {
 		fn(&v.Index)
 		fn(&v.Type)
 	case *Struct:
-		for i := range v.Members {
-			fn(&v.Members[i].Type)
+		if !reverse {
```

Comment:
The double negative and inverted order throws me off here. To me,
```go
if reverse {
    for i := ... {}
    break
}
```

is clearer than if/else with a double negative at the top.